src/Microsoft.Health.Fhir.CosmosDb/Features/Search/FhirCosmosSearchService.cs

```diff
@@ -47,11 +47,13 @@ public class FhirCosmosSearchService : SearchService
 
             if (searchOptions.IncludeTotal == TotalType.Accurate && !searchOptions.CountOnly)
             {
+                // TODO: Clone search options instead of mutating it.
```

Comment:
nit: maybe open a github issue and link to it from here?

Reply:
Good idea. Create #720 and added it to the comment in 884e2bb.